astropy/visualization/wcsaxes/coordinate_range.py

```diff
@@ -9,6 +9,8 @@
 
 # Algorithm inspired by PGSBOX from WCSLIB by M. Calabretta
 
+LONLAT = {'longitude', 'latitude'}
```

Comment:
This is fine, but I'm just curious why you used a set here (instead of tuple or list).

Reply:
For some reason I thought it would be faster but I realize now a tuple would be better, so I'll fix it next time I have the chance (though the difference is 33ns vs 39ns...)